src/jdk.incubator.code/share/classes/jdk/incubator/code/parser/OpParser.java

```diff
@@ -121,8 +124,7 @@
  */
 public final class OpParser {
 
-    static final TypeElement.ExternalizedTypeElement VOID =
-            new TypeElement.ExternalizedTypeElement("void", List.of());
+    static final TypeElement.ExternalizedTypeElement VOID = JavaType.VOID.externalize();
```

Comment:
This is a workaround: the parser uses a private void type -- but such a type is not handled by either the core type factory or the Java type factory. So injecting such type into the parsed model results into issues.